docs/julia/Runner.md

```diff
@@ -0,0 +1,119 @@
+# Julia Runner
+
+## Overview
+Julia Runner is one of the _benchmark runners_ described in [Architecture.md](../Architecture.md). Julia runner loads and runs _testing modules_, which in this case are modules in the sense of Julia language. 
+
+Every _testing module_ contains an implementation of some algorithm computing objective functions and their derivatives. Every objective should be supported by both the runner and the module to be benchmarked.
+
+A module doesn't have to support all of the objective types. If a user asks the runner to benchmark an unsupported objective, the runner prints an error and stops.
+
+Every module, for every objective it supports, exports a parameterless factory function that returns an object of the corresponding instantiation of generic [`Test{Input, Output}`](./Modules.md#itest-implementation) type.
+
+## Supported Objective Types
+Currently supported objective types:
+     
+| Full Name | Short Name |
+| -- | -- |
+| Gaussian Mixture Model Fitting | GMM |
+| Bundle Adjustment| BA |
+| Hand Tracking | Hand |
+| Long short-term memory | LSTM |
+
+
+## Command Line
+
+```powershell
+julia --project=/path/to/repo/JuliaProject.toml --optimize=3 Runner.jl test_type module_path input_filepath output_dir minimum_measurable_time nruns_F nruns_J time_limit [-rep]
+```
+
+ - `test_type` - The short type name of the objective function. It may also be equal to "Hand-Complicated" that designates the complicated case of the "Hand" objective, where the variable U is considered (see [srajer-autodiff-screen.pdf](../../Documents/srajer-autodiff-screen.pdf), page 5).
```

Comment:
Keeping a style `The` should be replaced by `the`, I think

Reply:
Fixed.